packages/embed/src/utils/build-iframe-src.ts

```diff
@@ -19,17 +20,21 @@ const typesToEmbed: Record<EmbedType, string> = {
   'side-tab': 'popup-side-panel',
 }
 
-const mapOptionsToQueryParams = (type: EmbedType, embedId: string, options: UrlOptions): Record<string, any> => ({
-  'typeform-embed-id': embedId,
-  'typeform-embed': typesToEmbed[type],
-  'typeform-source': options.source,
-  'typeform-medium': options.medium,
-  'typeform-medium-version': options.mediumVersion,
-  'embed-hide-footer': options.hideFooter ? 'true' : undefined,
-  'embed-hide-headers': options.hideHeaders ? 'true' : undefined,
-  'embed-opacity': options.opacity,
-  'disable-tracking': options.disableTracking ? 'true' : undefined,
-})
+const mapOptionsToQueryParams = (type: EmbedType, embedId: string, options: UrlOptions): Record<string, any> => {
+  const transferredParams = getTransitiveSearchParams(options.transferableUrlParameters as string[])
```

Comment:
`options.transferableUrlParameters` should be `options.transitiveSearchParams`

I suggest to also rename `transferredParams` to `transitiveParams`.